src/util/unread-notifications.js

```diff
@@ -0,0 +1,7 @@
+import { remoteFunction } from 'src/util/webextensionRPC'
+
+const getUnreadCount = remoteFunction('getUnreadCount')
+
+export default async function unreadNotifNumber() {
```

Comment:
I don't see any point to this module. Every usage could be replaced with a remoteFunction call.

Reply:
Here we are using this function in two places, `popup` and `overview`. So I was thinking just to make a util function that can use at both places. What you think now?

Reply:
Yes I saw it's being used in both scripts, but this is the same situation with a number of our "remote functions" exposed from the background script. If we do it for this one, then it would only make sense to be consistent and do it for the others. But again, I don't see any reason to - that's what the `util/webextensionRPC` module exists for. Sadly that module will lose typings for all the remote functions right now, but that would be a good enhancement later.